src/platform/orchestrator/src/MpiServer.cpp

```diff
@@ -594,16 +594,24 @@ static bool ReadRequest(int connfd, Request& request)
         {
             if (contentLength != (bytesRead = read(connfd, buffer, contentLength)))
             {
-                OsConfigLogError(PlatformLog::Get(), "Failed to read complete HTTP body: Content-Length %d, bytes read %d", contentLength, (int)bytesRead);
+                if (IsFullLoggingEnabled())
+                {
+                    OsConfigLogError(PlatformLog::Get(), "Failed to read complete HTTP body: '%s' Content-Length %d, bytes read %d, body ' %.*s'", uri, contentLength, static_cast<int>(bytesRead), static_cast<int>(bytesRead), buffer);
```

Comment:
>"Failed to read complete HTTP body: '%s' Content-Length %d, bytes read %d, body ' %.*s'",  [](http://example.com/codeflow?start=57&length=90)

Why not just:

Same as before +

"Failed to read complete HTTP body for '%s'...."

Reply:
Or even w/o ''

"%s: failed to read complete HTTP body...."

where %s is the uri